src/main/java/com/fasterxml/jackson/databind/cfg/MapperConfigBase.java

```diff
@@ -867,6 +868,21 @@ public Boolean getDefaultMergeable(Class<?> baseType) {
         return _configOverrides.getDefaultMergeable();
     }
 
+    // since 2.16
+    @Override
+    public JsonTypeInfo.Value getDefaultPolymorphicTypeHandling(Class<?> baseType)
+    {
+        ConfigOverride superOverride = _configOverrides.findOverride(baseType.getSuperclass());
```

Comment:
Ok this is unfortunately problematic in couple of ways:

1. It uses less specific (parent) information if both exist (should prefer specific one)
2. Assumes 2 levels of inheritance (could have more)
3. For interfaces `getSuperClass()` returns `null` I think? I don't think this would work when base type is interface

It is also different from all other config overrides which only consider exact `baseType` passed. I think that'd be correct thing to do here -- except I suspect there was a problem wrt serialization side: on deserialization one usually specifies proper base type (to get to polymorphic subtype). But on serialization, unfortunately, the way things are "baseType" is probably actual implementation type.
So using just that might not work.

If so I am not sure if this can actually be made to work, without fixing more fundamental problem: Jackson really should find proper base type on serialization too and not rely on annotation inheritance (which is why things work at all wrt `@JsonTypeInfo`).